scripts/intfutil

```diff
@@ -1,25 +1,29 @@
 #! /usr/bin/python
 
-import swsssdk
-import sys
+import argparse
+import os
 import re
+import sys
 import types
-from tabulate import tabulate
+
 from natsort import natsorted
-from swsssdk import ConfigDBConnector
-from pprint import pprint
+from tabulate import tabulate
+from utilities_common import constants
+from utilities_common import multi_asic as multi_asic_util
 from utilities_common.intf_filter import parse_interface_in_filter
 
-import os
-
 # mock the redis for unit test purposes #
 try:
-    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
+    if os.environ["UTILITIES_UNIT_TESTING"] >= "2":
         modules_path = os.path.join(os.path.dirname(__file__), "..")
         tests_path = os.path.join(modules_path, "tests")
         sys.path.insert(0, modules_path)
         sys.path.insert(0, tests_path)
         import mock_tables.dbconnector
+        if os.environ["UTILITIES_UNIT_TESTING"] == "3":
```

Comment:
i think we need to keep this as '2': but maybe we can introduce a new environmental variable, "UTILITIES_UNIT_TESTING_TOPOLOGY"="multi-asic"